pybliometrics/scopus/tests/test_SerialTitle.py

```diff
@@ -20,9 +25,21 @@ def test_aggregation_type():
 
 
 def test_citescoreyearinfolist():
-    expected1 = [(2022, 5.1), None]
-    assert_equal(sofwarex.citescoreyearinfolist, expected1)
-    assert_equal(oecd.citescoreyearinfolist, None)
+    expected_year = 2022
+    expected_citeScore = 5.1
+    expected_previous_year = None
+
+    assert_equal(sofwarex.citescoreyearinfolist[0][0], expected_year)
+    assert_equal(sofwarex.citescoreyearinfolist[0][1], expected_citeScore)
+    assert_equal(sofwarex.citescoreyearinfolist[1], expected_previous_year)
```

Comment:
Can we not test this all at once, i.e., by asking whether the return value is a namedtuple with the right keys and values? This would also test the keys of the namedtuples, not just the values